devtools-ci-index.yaml

```diff
@@ -1630,6 +1631,13 @@
         <<: *vault_defaults
         secrets: 
             - *quay-eclipse-che-credentials
+    triggers:
+        - github
+    scm:
+        - git:
+            url: https://{github_user}@github.com/{git_organization}/{git_repo}.git
+            branches:
+                - origin/cico-nightly-test
```

Comment:
This will not be "nightly" job, as it's going to be triggered by `github` trigger. Either we should change it to be triggered by [cron](https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.timed), or change the name to reflect what the job really does.
I'm for the variant to changing it to proper "nightly" job.

Reply:
It is just temporary solution. I am going to extend the  main script and launch this for debugging. If we get stable results i will return cron trigger.